src/valkey-cli.c

```diff
@@ -3017,6 +3024,8 @@ static void usage(int err) {
             "                     And define number of key elements to sample\n"
             "  --hotkeys          Sample keys looking for hot keys.\n"
             "                     only works when maxmemory-policy is *lfu.\n"
+            "  --hotkeys-count <n> Number of Sample keys looking for hot keys.\n"
+            "                     only works when maxmemory-policy is *lfu.\n"
```

Comment:
Maybe we can mention that `--hotkeys` is equivalent to `--hotkeys-count 16`?

Also we can avoid "number of Sample" because it samples all the keys in the database. N is the number of hot keys to report.

```suggestion
            "  --hotkeys          Sample keys looking for hot keys.\n"
            "                     Only works when maxmemory-policy is *lfu.\n"
            "                     This is equivalent to --hotkeys-count 16.\n"
            "  --hotkeys-count <n> Sample keys looking for the n most hot keys.\n"
            "                     Only works when maxmemory-policy is *lfu.\n"
```